packages/cli/src/commands/org/resources/pull.ts

```diff
@@ -33,6 +34,29 @@ import {SnapshotFactory} from '../../../lib/snapshot/snapshotFactory';
 import {confirmWithAnalytics} from '../../../lib/utils/cli';
 import {spawnProcess} from '../../../lib/utils/process';
 
+const PullCommandStrings = {
```

Comment:
Yeah, I understand your point, but when I was reading back the code, I found it was not really 'nice' to see where to split, especially because it's not only static strings, but there are params slipping in, and, moreover, the strings are on multiple lines.

I think we should not necessarily do this for 'all the message' but, when it gets big, I think it'll also help the API writer team.

To alleviate the 'newcomer' effect, doing good naming of the variables would help I think.

Reply:
@y-lakhdar , tho, what do you think?

I can see it goes both way really

Reply:
Don't really mind, but I find the new approach to make the code more breathable